src/IAM/Infrastructure/Persistence/MySQL/Migrations/Version20251217102810.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Twitter\IAM\Infrastructure\Persistence\MySQL\Migrations;
```

Comment:
@adnlv (low) Most of the folder names are in singular not in plural.
```suggestion
namespace Twitter\IAM\Infrastructure\Persistence\MySQL\Migration;
```